server/src/test/java/org/elasticsearch/index/IndexVersionTests.java

```diff
@@ -151,9 +150,7 @@ public void testMax() {
         }
     }
 
-    @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA)
-    @AwaitsFix(bugUrl = "believe this fails because index version has not yet been bumped to 9.0")
-    public void testMinimumCompatibleVersion() {
```

Comment:
This was an outdated awaitsfix, we can simply remove it. I went ahead and renamed the test method as well for clarity